flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1107,20 +1107,12 @@ static void altitude_hold_desired(ManualControlCommandData * cmd, bool flightMod
 
 static void set_loiter_command(ManualControlCommandData *cmd)
 {
-	const float CMD_THRESHOLD = 0.5f;
-	const float MAX_SPEED     = 3.0f; // m/s
-
 	LoiterCommandData loiterCommand;
-	loiterCommand.Forward = (cmd->Pitch > CMD_THRESHOLD) ? cmd->Pitch - CMD_THRESHOLD :
-	                        (cmd->Pitch < -CMD_THRESHOLD) ? cmd->Pitch + CMD_THRESHOLD :
-	                        0;
-	// Note the negative - forward pitch is negative
-	loiterCommand.Forward *= -MAX_SPEED / (1.0f - CMD_THRESHOLD);
-
-	loiterCommand.Right = (cmd->Roll > CMD_THRESHOLD) ? cmd->Roll - CMD_THRESHOLD :
-	                      (cmd->Roll < -CMD_THRESHOLD) ? cmd->Roll + CMD_THRESHOLD :
-	                      0;
-	loiterCommand.Right *= MAX_SPEED / (1.0f - CMD_THRESHOLD);
+
+	loiterCommand.Pitch = cmd->Pitch;
```

Comment:
Ok cool. One thing I'm not sure there is a good solution for. I think analyzing log files with this might be a bit tricky since we aren't really exposing all the internal things that alter the control scheme. Not sure if there is a good solution to this.